### DISCUSSION TITLE
Output Directory

### DISCUSSION THREAD

**Discussion by tG-nest** [WinMerge/winmerge#918] [Q&A]
Hi - I'm looking for assistance in streamlining output generated from winmerge when comparing files (e.g., executed code) for audit. In particular, i'm taking an entire directory with subfolders and comparing month over month which expected minor differences (e.g., mismatched file names). The issue i need support with is how to efficiently save down the generated html outputs based upon the several compares I perform. Is there a more efficient way to save down the compiled output without having to generate a winmerge html report individually for each result?

**Comment by sdottaka** [comment]
I may not have understood exactly what you want, but if you enable the checkbox for "Include file comparison report" when creating folder comparison report, the folder comparison report and the file comparison report for files with differences will be created at the same time. This eliminates the need to manually select files one by one to create a file comparison report.

![image](https://user-images.githubusercontent.com/98126/129428583-95582485-4b73-414d-a8fd-2ea103d0f7e9.png)
